server/listen_windows.go

```diff
@@ -0,0 +1,16 @@
+// +build windows
+
+package server
+
+import (
+	"net"
+
+	"github.com/Microsoft/go-winio"
+)
+
+// Listen opens the network address for the server. Expects the config.Listen address.
+//
+// This is a platform specific wrapper.
+func Listen(network, address string) (net.Listener, error) {
+	return winio.ListenPipe(address, nil)
```

Comment:
It feels strange to ignore `network` when it's [set to `unix`][1].  Perhaps we should drop `network` from `Listen`'s arguments, and have the per-OS implementations decide what sort of connection to make?

[1]: https://github.com/kubernetes-incubator/cri-o/pull/1655/files#diff-702431da3a2e35392cf253955d15f774R499

Reply:
or i'll add a switch here